client.go

```diff
@@ -56,6 +56,11 @@ type Client interface {
 	// partition. Offline replicas are replicas which are offline
 	OfflineReplicas(topic string, partitionID int32) ([]int32, error)
 
+	// RefreshBrokers takes a list of addresses to be used as seed brokers.
+	// Existing broker connections are closed and the updated list of seed brokers
+	// will be used for the next metadata fetch.
+	RefreshBrokers(addrs []string) error
+
```

Comment:
I'm not sure how Sarama folks are adding new features, but this looks like it's a breaking change if someone is implementing the interface 🤔 

not sure why someone would do that, but the interface is public 🤷 

we could add a trick that I saw in the golang code, to add a private method, so no apps could implement the interface

```
    // A private method to prevent users implementing the
    // interface and so future additions to it will not
    // violate Go 1 compatibility.
    private()
```

found the blog post: https://blog.golang.org/module-compatibility

Reply:
Good idea, i'll add a dummy `private()` method to the interface

Reply:
https://github.com/Shopify/sarama/pull/1785

Reply:
😞 this wasn't a good idea at the end :/ the initial intention of `private()` is if the user needs to use an interface but don't want other people to implement it, and this is not the case for this interface.